swath_projector/reproject.py

```diff
@@ -69,7 +69,7 @@ def reproject(
 
     # Loop through each dataset and reproject
     logger.debug('Using pyresample for reprojection.')
-    outputs = resample_all_variables(
+    outputs, failed_variables = resample_all_variables(
```

Comment:
what if you renamed outputs to `resampled_variables`?  or something similar? `outputs` is kinda generic.

Reply:
unless it's `output_variables`?

Reply:
I liked the suggestion to rename to `resampled_variables`. [946b3a3](https://github.com/nasa/harmony-swath-projector/pull/27/commits/946b3a30b3fbc3924a25c2096623d163b3944ac1)